talk/owt/sdk/ic/backgroundblur.cc

```diff
@@ -0,0 +1,85 @@
+// Copyright (C) <2021> Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "backgroundblur.h"
+
+#include <exception>
+#include <limits>
+
+#include "libyuv/convert.h"
+#include "opencv2/imgproc.hpp"
+#include "third_party/webrtc/api/video/i420_buffer.h"
+#include "third_party/webrtc/rtc_base/logging.h"
+
+#include "selfiesegmentationmodel.h"
+
+namespace owt {
+namespace ic {
+
+BackgroundBlur::BackgroundBlur(InferenceEngine::Core& core) : model(core) {}
+
+bool BackgroundBlur::SetParameter(const std::string& key,
+                                  const std::string& value) {
+  if (key == "model_path") {
+    return model.LoadModel(value);

```

Comment:
Is this a blocking call? Do you expect caller do this on a task queue?

Reply:
It is blocking. When the function returns, the model should be ready for inference. The caller do the task queue / threading if they don't want to wait.

Reply:
Need to clearly call this out in your API doc